inc/reservation.class.php

```diff
@@ -901,303 +937,60 @@ static function computePeriodicities ($begin, $end, $options = []) {
    }
 
 
-   /**
-    * Display for reservation
-    *
-    * @param $ID     ID a the reservation item (empty to show all)
-    * @param $date   date to display
-   **/
-   static function displayReservationDay($ID, $date) {
```

Comment:
Method was public. Should it be deprecated ?

Reply:
i think not. i'm not aware about a plugin using this part.

Reply:
adding an entry to changelog